web/src/components/pages/dashboard/challenge/weekly-challenge-board.tsx

```diff
@@ -44,86 +47,81 @@ export default function WeeklyChallengeBoard({
+                </LocaleLink>
                 <CircleProgress
-                  className="listen-bar"
+                  className="listen-bar progress-desktop"
                   value={user.listen}
                   denominator={user.listen_total}
                   strokeW={4}
                   radius={66}
                 />
-              ) : (
-                <div className="fraction listen">
-                  <div className="numerator">{user.listen}</div>
-                  <div className="denominator"> / {user.listen_total}</div>
-                </div>
-              )}
+                <CircleProgressSimple
+                  numerator={user.listen}
+                  denominator={user.listen_total}
+                  className="listen-bar progress-mobile"
+                />
+              </div>
             </div>
           </div>
           <div className="content-row">
-            <div className="column">
+            <div className="column avatar-container">
               <TeamAvatar team={teamToken} />
               <p>Team</p>
             </div>
-            <div className="container team">
-              <p className="team-text">
-                Win a prize by being the team with the highest sign up rate
-              </p>
-              <div className="v-divider" />
-              <p className="team-invite">
-                The is the percentage of team invites that have been accepted
-                out of the current total sent.
-              </p>
-            </div>
-            <div className="container invite">
-              {!isNarrow ? (
+            <div className="column challenge-stats team-stats">
+              <div className="container team">
+                <p className="team-text">
+                  Win a prize by being the team with the highest sign up rate
+                </p>
+                <div className="v-divider" />
+                <p className="team-invite">
+                  The is the percentage of team invites that have been accepted
+                  out of the current total sent.
+                </p>
+              </div>
+              <div className="container invite">
                 <CircleProgress
-                  className="team-bar"
+                  className="team-bar progress-desktop"
                   value={team.invite / team.invite_total}
                   strokeW={4}
                   radius={66}
                 />
-              ) : (
-                <React.Fragment>
-                  <div className="team-invite-percentage">
-                    <span>{team.invite / team.invite_total}</span>%
-                  </div>
-                  <div className="divider"></div>
-                </React.Fragment>
-              )}
-              <p className="team-invite-total">
-                of {team.invite_total} invites
-              </p>
+                <CircleProgressSimple
+                  numerator={(100 * team.invite) / team.invite_total}
```

Comment:
Nitpick: we should update these props to match the signature of `CircleProgress`.